test/unit/spec/Registration_spec.js

```diff
@@ -652,6 +652,41 @@ function (Q, _, $, OktaAuth, Backbone, Util, Expect, Beacon, RegForm, RegSchema,
           expect(setting.registration.postSubmit).toHaveBeenCalled();
         });
       });
+      itp('calls postSubmit call onSuccess assert username is same as email', function () {
```

Comment:
nit: May want to change the name of the test to something like `asserts username is same as email and verifies  text on registration complete view`